unity-environment/Assets/ML-Agents/Scripts/CoreBrainInternal.cs

```diff
@@ -513,7 +513,7 @@ public void OnInspector()
             pixels = 1;
         else
             pixels = 3;
-        float[,,,] result = new float[batchSize, width, height, pixels];
```

Comment:
Please also update the documentation for the method: BatchVisualObservations, using the doxygen-friendly format and ensure that we comment on the dimensions of the return variable.